getting-started-with-api/java-sdk/src/main/java/DownloadPseudo.java

```diff
@@ -0,0 +1,71 @@
+import com.smartling.api.files.v2.FilesApi;
+import com.smartling.api.files.v2.FilesApiFactory;
+import com.smartling.api.files.v2.pto.DownloadTranslationPTO;
+import com.smartling.api.files.v2.pto.RetrievalType;
+import com.smartling.api.v2.authentication.AuthenticationApi;
+import com.smartling.api.v2.authentication.AuthenticationApiFactory;
+import com.smartling.api.v2.client.ClientFactory;
+import com.smartling.api.v2.client.DefaultClientConfiguration;
+import com.smartling.api.v2.client.auth.Authenticator;
+import com.smartling.api.v2.client.auth.BearerAuthSecretFilter;
+
+import com.smartling.api.v2.client.exception.RestApiRuntimeException;
+import org.apache.commons.io.IOUtils;
+import java.io.InputStream;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class DownloadPseudo
+{
+    public static void main( String[] args ) throws Exception
+    {
+        // Read command-line arguments
+        if (args.length != 2) {
+            System.out.println("Usage: mvn -X exec:java -Dexec.mainClass=\"DownloadPseudo\" -Dexec.args=\"<fileUri> <localeId>\"");
+            System.out.println("E.g.: mvn -X exec:java -Dexec.mainClass=\"DownloadPseudo\" -Dexec.args=\"strings.json fr-FR\"");
+            System.out.println("Takes two command-line parameters: ");
+            System.out.println("  fileUri: URI in Smartling of the file to be downloaded");
+            System.out.println("  localeId: any locale ID that is defined in the Smartling project");
+            System.exit(1);
+        }
+        String fileUri = args[0];
+        String localeId = args[1];
+
+        // Read authentication credentials from environment
+        String userId = System.getenv("DEV_USER_IDENTIFIER");
+        String userSecret = System.getenv("DEV_USER_SECRET");
+        String projectId = System.getenv("DEV_PROJECT_ID");
```

Comment:
Is there something more universal: read from command line params if no then try to read from env. variables

Reply:
I think the idea here is that user doesn't pass sensitive information in command line, but rather using environment variable. Otherwise it doesn't make sense to mix arguments and environment variables

Reply:
I'm using the same approach of environment variables throughout the docs and tutorials, so the user should be familiar with it.